.golangci.yml

```diff
@@ -1,5 +1,31 @@
-# Run only staticcheck for now. Additional linters will be enabled one-by-one.
+---
+
+run:
+  deadline: 5m
+
+output:
+  sort-results: true
+
 linters:
   enable:
-  - staticcheck
-  disable-all: true
+    - gofumpt
+    - goimports
+    - revive
+    - misspell
+
+issues:
+  max-same-issues: 0
+  exclude-rules:
+    - path: _test.go
+      linters:
```

Comment:
These linters don't seem to be enabled, no?

Reply:
`golangci-lint` has a list of default linters that are enabled by default, unless being disabled explicitly or implicitly. in the previous config they were disabled implicitly by using `disable-all: true` in [this line](https://github.com/prometheus/client_golang/pull/1056/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9L5)

the [default linters](https://golangci-lint.run/usage/linters/#enabled-by-default) list is:

  - deadcode
  - errcheck
  - gosimple
  - govet
  - ineffassign
  - staticcheck
  - structcheck
  - typecheck
  - unused
  - varcheck

since in the main prometheus repo they left them enabled implicitly i did the same too but i understand that it could reduce readability. 

do you want me to enable them in the config explicitly or leave them as they are?

Reply:
Having them explicit would be amazing. Also, this would prevent any unwanted issues, if we upgrade golangci-lint and decide to change their default preset.